BraveRewardsUI/Extensions/NumberExtensions.swift

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+extension Double {
+  /// Format of which BAT should be presented to user.
+  /// Similar method also exists for Strings.
+  var BATDisplayFormat: String {
```

Comment:
Slight naming change I think here. It sounds like this should be returning some String format (like `"%.1"`) and not the actual BAT string.  Perhaps something like `formattedBATDisplayString`?

It actually makes me wonder if we should just make a `BATValue` struct or something that takes a double/string/probi and lets us pull out whatever type we want?  Then we won't have to make extensions straight on `String`/`Double`

For grants we're actually going to get a `String` but it won't be in the format of like "30.0", it'll be a probi string which will be `"30000000000000000000"` (30 * 10^18)

Also lets us isolate tests around BAT values to one struct which is a plus. 

Let me know what you think about that.

Reply:
I like it, especially if there's more than just simple String and Double formats coming.
I will refactor this tomorrow